Dockerfile

```diff
@@ -16,6 +16,7 @@ COPY . .
 ARG version
 ENV version=${version}
 RUN ./scripts/build-engine.sh
+RUN ./scripts/build-tools.sh
```

Comment:
We shouldn't use our internal dev tools in the official docker image.

Reply:
This is not an internal dev tools. It’s a generic tool also use in dev.